public/style/reply.css

```diff
@@ -31,19 +31,19 @@
   border: 0em;
 }
 
-#firstPP {
+#firstProfilePicture {
   z-index: 1;
 }
 
-#secondPP {
+#secondProfilePicture {
```

Comment:
What is a secondProfilePicture

as far as i know every user gets one profile picture 🤔